cluster/environments/azure-stateful-keyvault/README.md

```diff
@@ -0,0 +1,3 @@
+# azure-stateful-keyvault
+
+The `azure-stateful-keyvault` environment restores a single production level AKS cluster from a Velero backup using the Velero Terraform module.
```

Comment:
Also, can we include more instructions here, or in a link out to the `common/velero` component where we describe the process for doing a restore?